### ISSUE TITLE
Changing size of the legend

### ISSUE THREAD

**Issue by Micro-Sandworms** [kumu/docs#28]
Hi,

Thank you for this amazing program. Is there any way to change the font size or the overall size of the legend?

Regards

**Comment by danprince** [comment]
In the name of Shai-Hulud, I am afraid not!

We have a color legend control: https://docs.kumu.io/guides/controls/color-legend-control and many of our [interactive controls](https://docs.kumu.io/guides/controls/controls-reference#interactive-controls) support a range of properties (like `font-size`) if you're comfortable in the advanced editor.

It's possible that we might support a `legend-scale` property in the future, which would rescale the legend (e.g. `legend-scale: 2` would make it twice as big as normal). Would that solve your usecase?

**Comment by Micro-Sandworms** [comment]
Certainly, the legend-scale option would add spice to it, and solve my use case.

**Comment by danprince** [comment]
Features are the product killer, but if/when we get around to adding this property, I'll let you know here.
